lab4.md

```diff
@@ -2,6 +2,13 @@
 
 <img src="https://github.com/openfaas/media/raw/master/OpenFaaS_Magnet_3_1_png.png" width="500px"></img>
 
+Before starting this lab, create a new folder for your files:
+
+```
+$ mkdir -p lab4 \
```

Comment:
Here `hello-openfaas` is reused from Lab 3

Reply:
Yes. `cp -r lab3/hello-openfaas` and `cp lab3/hello-openfaas.yml` could also work